apps/api/validators.py

```diff
@@ -1,12 +1,384 @@
-from django.core.exceptions import ValidationError
+'''Validates if the file given is a csv file'''
 
 import csv
+from re import match
+from django.core.exceptions import ValidationError
+
+
+class ColumnError(Exception):
```

Comment:
Currently just inherits from Exception and passes. Every time it is raised, it uses the same message with the same parameters : current row number, expected column number, and row’s number of column. Will be troublesome to find and modify all if something changes.

Possible solution (using same variable names and message):

	class ColumnError(Exception):
		def __init__(self, line_number, col_number, len_row):
			msg: ‘Row: {}. Expected: {} columns. Found {} columns’.format(line_number, col_number, len_row)
			super(ColumnError, self).__init__(msg)

EX. raise ColumnError(8, 4, 2)
Row: 8. Expected: 4 columns. Found 2 columns